src/Simulation/Simulators.Tests/StackTraceTests.cs

```diff
@@ -26,6 +26,23 @@ public StackTraceTests(ITestOutputHelper output)
             this.output = output;
         }
 
+        [Fact]
+        public void AllocateQubit2Test()
+        {
+            using (var sim = new QuantumSimulator())
+            {
+                try
+                {
+                    QVoid res = AllocateQubit2.Run(sim).Result;
+                }
+                catch (AggregateException ex)
+                {
+                    Assert.True(ex.InnerException is ExecutionFailException);
+                    // TODO: Need more checks here - stack frames and line numbers
```

Comment:
What was the reason to not add the additional checks for the stack frames?

Reply:
As I mentioned in the description, more fixes are needed for the stack frames to be correct. If I add additional checks now they will fail.
I'm currently working on better C#-Q# frame matching. When this is done I'll add more checks.